content/docs/studio/user-guide/prepare-your-repositories.md

```diff
@@ -4,7 +4,7 @@
 
 </admon>
 
-# Prepare Your Repositories
+# Prepare your Repositories
```

Comment:
I'd suggest using `Your` instead of `your` for consistency with the rest of the pages.